homework_1/dzOne.java

```diff
@@ -0,0 +1,14 @@
+import java.util.Scanner;
+
+public class dzOne {
+  
+    public static void main(String[] args) {
+          
+        Scanner in = new Scanner(System.in);
+		System.out.print("Input a number: ");
```

Comment:
Scanner works as input-output stream, it's not a way to get command line arguments for java program